src/main/java/net/laboulangerie/laboulangeriecore/houses/housewand/HouseWandListener.java

```diff
@@ -0,0 +1,87 @@
+package net.laboulangerie.laboulangeriecore.houses.housewand;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer;
+
+public class HouseWandListener implements Listener {
+
+    public static Location firstPos;
+    public static Location secondPos;
+
+    private int getBlocksCount() {
```

Comment:
isn't it `abs(firstPos.x-secondPos.y)*abs(firstPos.y-secondPos.y)*abs(firstPos.z-secondPos.z)` ?